static/js/components/HeaderContent.jsx.template

```diff
@@ -2,31 +2,67 @@ import React from "react";
 import { Link } from "react-router-dom";
 
 import Box from "@material-ui/core/Box";
+import { makeStyles } from "@material-ui/core/styles";
 import ProfileDropdown from "./ProfileDropdown";
 import Logo from "./Logo";
 import Notifications from "./Notifications";
-import styles from "./Main.css";
 
-const HeaderContent = () => (
-  <div className={styles.topBannerContent}>
-    <div style= {%- raw -%} {{display: "inline-flex", flexDirection: "row" }} {%- endraw -%} >
-      <div className={styles.logos}>
-        {%- for logo in app.logos %}
-        <div className={styles.logoContainer}>
-          <Logo src="{{ logo.src }}" altText="{{ logo.alt_text }}" />
+const useStyles = makeStyles((theme) => ({
+  topBannerContent: {
+    position: "relative",
+    paddingLeft: "1em",
+    paddingRight: "1em",
+    paddingTop: "1.5em",
+  },
+  title: {
+    textDecoration: "none",
+    color: "white",
+    paddingLeft: "0.4em",
+    fontSize: "200%",
+    fontWeight: "bold",
+    verticalAlign: "middle",
+  },
+  logos: {
+    display: "none",
+    // Only show on bigger screens
+    [theme.breakpoints.up("sm")]: {
+      display: "inline-flex",
+      flexDirection: "row",
+    },
+  },
+  logoContainer: {
+    marginTop: "-0.5em",
+    height: "50px",
+    borderRight: "1px solid white",
+    padding: "0 0.75rem",
+    "&:last-of-type": {
+      borderRight: "none",
+      paddingRight: "0",
+    },
+  },
+}));
+const HeaderContent = () => {
+  const classes = useStyles();
+  return (
+    <div className={classes.topBannerContent}>
+      <div style= {%- raw -%} {{display: "inline-flex", flexDirection: "row" }} {%- endraw -%} >
```

Comment:
I think

```
<div style="display: inline-flex, flexDirection: row">
```

should work just as well.

Reply:
I think it's like this because ESLint doesn't like the passing strings into the style thing.